library/psa_crypto.c

```diff
@@ -4317,29 +4381,31 @@ psa_status_t psa_cipher_finish( psa_cipher_operation_t *operation,
     else
     {
         status = PSA_ERROR_BUFFER_TOO_SMALL;
-        goto error;
+        goto exit;
```

Comment:
Not necessary anymore, probably better to remove it.

Reply:
Done